### ISSUE TITLE
TL Ready to be published pop-up

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cvs.two#427]
**Original report on BitBucket by Taina Jääskeläinen.**

----------------------------------------

Language-specific version history for TLs is produced at the time the TL admins clicks on ‘Ready to be published’. 

If the language content for that vocabulary has been published previously, TL admins get a pop-up where they have :

* change log
* comparison table
* editable ‘Version notes’ and ‘Version history’ fields
* non-editable version number copied from SL at the time the SL status is changed to ‘Ready to be translated’. 

‌

* Change log

    * The system automatically copies information on deprecated and added codes here from SL. The information on deprecations and additions is copied from the SL change log after the SL admin has changed the status to ‘Ready to be translated’.
    * The other changes in the change log come from change types entered into edit CV and edit code pop-ups in the TL \(just as now\).
    * The change log information is copied into ‘Version history’ field.
    

Basically this is the same pop-up that the TLs have now at the time when they change the status to Published, except that the version number is non-editable and ‘code deleted’ is now ‘code deprecated’. 

‌
